src/components/Firestore/Requests/RequestsCard/Table/TableRow.tsx

```diff
@@ -41,7 +41,7 @@ function getTableRowRequestData(
 ): TableRowRequestData {
   const { rulesContext, outcome } = request;
   // time * 1000 converts timestamp units from seconds to millis
-  const requestDate = new Date(rulesContext.time);
+  const requestDate = new Date(request.time);
```

Comment:
Please use the destructuring assignment above on L42 and remove the outdated comment on L43. LGTM otherwise.

Reply:
ah, good point @yuchenshi! I'll do that now